api/envoy/config/filter/http/adaptive_concurrency/v2alpha/adaptive_concurrency.proto

```diff
@@ -61,4 +62,7 @@ message AdaptiveConcurrency {
     GradientControllerConfig gradient_controller_config = 1
         [(validate.rules).message.required = true];
   }
+
+  // If set to true, the adaptive concurrency filter will operate as a pass-through filter.
```

Comment:
What does true mean here exactly? This is a uint32t value. I think you probably want to add a `RuntimeFeatureFlag` message or similar per the previous discussion?